src/services/static-snapshot-service.ts

```diff
@@ -1,20 +1,113 @@
+import * as bodyParser from 'body-parser'
+import * as cors from 'cors'
+import * as express from 'express'
+import {Server} from 'http'
+import * as puppeteer from 'puppeteer'
+import * as walk from 'walk'
 import logger from '../utils/logger'
+import {agentJsFilename} from '../utils/sdk-utils'
 import {StaticSnapshotOptions} from './static-snapshot-options'
 
+// Use this instead of importing PercyAgent - we only want the compiled version
+declare var PercyAgent: any
+
 export default class StaticSnapshotService {
+  private readonly app: express.Application
+  private readonly options: StaticSnapshotOptions
```

Comment:
```suggestion
  readonly options: StaticSnapshotOptions
```

let's not make this private. It seems like a reasonable interface to offer publicly and will make testing easier.